### PR TITLE
Data normalizer: unify licenses and descriptions

### PR DESCRIPTION
**Pull request by SeverusYixin**
Update to normalize the following terms:

'Bsd 2-Clause' or 'Bsd-2-Clause' to 'BSD-2-Clause' 'Bsd 3-Clause' or 'Bsd-3-Clause' to 'BSD-3-Clause' 'Cc By 4.0', 'Cc-By 4.0', 'Cc-By-4.0', 'Creative Commons / Attribution 4.0 International (Cc By 4.0)', or 'Creative Commons Attribution 4.0 International' to 'CC-BY-4.0' 'Bio-Image Analysis' to 'Bioimage Analysis'
'Unclear' to 'Unknown'
Additionally, ensure that data in the 'description' field is not human-readable.

### CODE REVIEW

File: scripts/data_normalizer.py [hunk 1]
```diff
@@ -17,7 +17,9 @@
     "Cc-By-4.0": "CC-BY-4.0",
     "Creative Commons / Attribution 4.0 International (CC BY 4.0)": "CC-BY-4.0",
     "Creative Commons Attribution 4.0 International": "CC-BY-4.0",
-    "CC0 licence (some accessions are available under different license, if so the license is indicated as attribute on dataset)": "CC0",
+    "CC0 licence (some accessions are available under different license, if so the license is indicated as attribute on dataset)": "CC0-1.0",
+    "CC0": "CC0-1.0",
+    "PUBLIC DOMAIN": "CC0-1.0",
```
Comment (haesleinhuepf) [inline scripts/data_normalizer.py:22]: This is not correct in general. Have you checked the individual entry?

Comment (SeverusYixin) [reply scripts/data_normalizer.py:22]: Hmmm, I don't think I understood what you meant, did you mean that the way is wrong or something like "CC0 licence (some accessions are available under a different licence, if so the licence is indicated as an attribute on the dataset)" should not be CC0-1.0?

Comment (haesleinhuepf) [reply scripts/data_normalizer.py:22]: My comment refers to the line starting with "public domain". Not every license that's public domain is automatically cc0.

Comment (SeverusYixin) [reply scripts/data_normalizer.py:22]: Then we don't need to make changes to the PUBLIC DOMAIN, am I right?

Comment (haesleinhuepf) [reply scripts/data_normalizer.py:22]: No automatic changes at least.

Comment (SeverusYixin) [reply scripts/data_normalizer.py:22]: ok

**Comment by haesleinhuepf** [comment]
I did some manual curation of the yml file and will merge this.

Thanks @SeverusYixin !

**Comment by SeverusYixin** [comment]
No problem, after you merge I will check the material submitter again.

**Pull request merged**